src/helpers/MiscFunctions.cpp

```diff
@@ -722,6 +722,14 @@ int64_t configStringToInt(const std::string& VALUE) {
     return std::stoll(VALUE);
 }
 
+Vector2D configStringToVector2D(const std::string& VALUE) {
+    const auto SIZESTR  = VALUE.substr(VALUE.find(' ') + 1);
+    const auto SIZEXSTR = SIZESTR.substr(0, SIZESTR.find(' '));
+    const auto SIZEYSTR = SIZESTR.substr(SIZESTR.find(' ') + 1);
+
+    return Vector2D(std::stoll(SIZEXSTR), std::stoll(SIZEYSTR));
```

Comment:
this still won't parse `123,456` or `123, 456`

I want a function that parses all three:
 - `123 456`
 - `123,456`
 - `123 456`

Reply:
ok should be good now